examples/Huggingface_Transformers/Transformers_handler.py

```diff
@@ -0,0 +1,112 @@
+from abc import ABC
+import json
+import logging
+import os
+
+import torch
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class TransformersSeqClassifierHandler(BaseHandler, ABC):
+    """
+    Transformers text classifier handler class. This handler takes a text (string) and
+    as input and returns the classification text based on the serialized transformers checkpoint.
+    """
+    def __init__(self):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.initialized = False
+
+    def initialize(self, ctx):
```

Comment:
why not combine `self.initialize` func and constructor together? Do you expect user to initialize the class multiple times?

Reply:
Constructor and self.initialize func are combined.

Reply:
How about this one?

Reply:
Sure, modified as well. Thanks.

Reply:
@zhangguanheng66 The `initialize` method has a special purpose in TorchServe base model handlers. @mycpuorg Can you please describe the details for why it needs to be separate?